candidate/views_admin.py

```diff
@@ -1830,6 +1830,8 @@ def candidate_edit_view(request, candidate_id=0, candidate_we_vote_id=""):
     candidate_twitter_handle3 = request.GET.get('candidate_twitter_handle3', False)
     candidate_url = request.GET.get('candidate_url', False)
     candidate_contact_form_url = request.GET.get('candidate_contact_form_url', False)
+    contest_office_name = request.GET.get('contest_office_name', False)
+    district_name = request.GET.get('district_name', False)
```

Comment:
Wasn't sure if these should be `False` or `""`

Reply:
False is correct -- a "False" value is the signal we use to specify that a GET variable wasn't received.